trakt/tv.py

```diff
@@ -571,7 +571,6 @@ def images(self):
             self._images = data.get('images', {})
         yield self._images
 
-    @property
     def first_aired_date(self):
```

Comment:
Now youl have to call it as a function, not property. If you want to be able to set a property, use: `@first_aired_date.setter`

Reply:
I hate decorators >.<

Reply:
No prob. Revert the old `@property ` and add this:
```
  @first_aired_date.setter
  def first_aired_date(self, value):
        """Set python datetime object representation of the first_aired date of
        this :class:`TVEpisode`
        """
        self.first_aired = value
```

The @property functions as the original getter. And now setting the dat will also work through the new code.

Reply:
No dice.
```
Traceback (most recent call last):
  File "C:\Users\mhill\Downloads\PyTrakt-master\trakt_test.py", line 2, in <module>
    from trakt.calendar import MyShowCalendar
  File "C:\Users\mhill\Downloads\PyTrakt-master\trakt\calendar.py", line 6, in <module>
    from trakt.tv import TVEpisode
  File "C:\Users\mhill\Downloads\PyTrakt-master\trakt\tv.py", line 479, in <module>
    class TVEpisode(object):
  File "C:\Users\mhill\Downloads\PyTrakt-master\trakt\tv.py", line 574, in TVEpisode
    @first_aired_date.setter
NameError: name 'first_aired_date' is not defined
```

Any ideas?